server/controllers/routes/admin/login.js

```diff
@@ -0,0 +1,41 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
```

Comment:
also here You can make destructure here to get sign direct
```suggestion
const { sign } = require('jsonwebtoken');
```